tests/vm_networking_test.go

```diff
@@ -159,8 +165,43 @@ var _ = Describe("Networking", func() {
 			case "InboundVM":
 				addr = inboundVM.Status.Interfaces[0].IP
 			}
-			cmdCheck = fmt.Sprintf("ping %s -c 1 -w 5\n", addr)
+
+			By("checking br1 MTU inside the pod")
+			vmPod := tests.GetRunningPodByLabel(outboundVM.Name, v1.DomainLabel, tests.NamespaceTestDefault)
+			output, err := tests.ExecuteCommandOnPod(
+				virtClient,
+				vmPod,
+				vmPod.Spec.Containers[0].Name,
+				[]string{"ip", "address", "show", "br1"},
+			)
+			log.Log.Infof("%v", output)
+			Expect(err).ToNot(HaveOccurred())
+			// the following substring is part of 'ip address show' output
+			expectedMtuString := fmt.Sprintf("mtu %d", expectedMtu)
+			Expect(strings.Contains(output, expectedMtuString)).To(BeTrue())
```

Comment:
out of curiosity if we didn't supply mtu size over dhcp, what do you know what this value used to be?  I just want to make sure the mtu sizes don't happen to match just by chance.

Reply:
I could think of two scenarios that would prove the mechanism works as intended.

1. Somehow change pod network MTU to a unique value (1234?), then boot VM and check eth0 MTU is the same unique value. Sadly, we don't have a way to change pod network MTU in runtime. The closest we have is different MTU depending on used network provider; but it wouldn't fit functional test requirements.

2. Somehow disable DHCP server, boot VM, check and remember its MTU; re-enable DHCP server, reboot VM, check eth0 MTU is now different (configured by DHCP server). I don't think we have a way to disable / enable DHCP server either.

Any other options that would work that you can think of?